packages/xo-web/src/xo-app/pool/tab-advanced.js

```diff
@@ -396,6 +400,20 @@ export default class TabAdvanced extends Component {
                       <Toggle value={pool.auto_poweron} onChange={this._onChangeAutoPoweron} />
                     </td>
                   </tr>
+                  <tr>
+                    <th>{_('migrationCompression')}</th>
+                    <td>
+                      <Tooltip
+                        content={isMigrationCompressionAvailable ? _('migrationCompressionDisabled') : undefined}
+                      >
+                        <Toggle
+                          value={pool.migrationCompression}
+                          onChange={this._onChangeMigrationCompression}
+                          disabled={isMigrationCompressionAvailable}
+                        />
+                      </Tooltip>
```

Comment:
The toggle is *disabled* when the compression is *available*?
```suggestion
                      <Tooltip
                        content={isMigrationCompressionAvailable ? undefined : _('migrationCompressionDisabled')}
                      >
                        <Toggle
                          value={pool.migrationCompression}
                          onChange={this._onChangeMigrationCompression}
                          disabled={!isMigrationCompressionAvailable}
                        />
                      </Tooltip>
```

Reply:
Indeed, I forgot to change this after the modification of ``` isMigrationCompressionAvailable```